hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OmMetadataManagerImpl.java

```diff
@@ -805,51 +805,79 @@ public boolean isBucketEmpty(String volume, String bucket)
     return deletedKeys;
   }
 
+  /**
+   * @param userName volume owner, null for listing all volumes.
+   * @throws IOException
```

Comment:
Please do not add `@throws` without description.  Either omit it, or add description.

Reply:
Thanks @adoroszlai for the comment.
Updated with omit.